.gitignore

```diff
@@ -143,3 +143,6 @@ wandb
 # method_comparison logs
 method_comparison/MetaMathQA/cancelled_results/
 method_comparison/MetaMathQA/temporary_results/
+
+.peft/
+test.py
```

Comment:
Let's remove these. You can set up a [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for these cases.